### ISSUE TITLE
[transaction] registerFirstUser is missing req on _verified update 

### ISSUE THREAD

**Issue by geminigeek** [payloadcms/payload#3664]
### Link to reproduction

na

### To Reproduce

create a fresh project the first user its not set as verified 

### Describe the Bug

registerFirstUser is missing req on _verified update ,  this is the line where the issue is, this query is missing req

https://github.com/payloadcms/payload/blob/27589482dd20377916ab7ad67af0b4a46df0b92a/packages/payload/src/auth/operations/registerFirstUser.ts#L70

this is current code 
```
 if (verify) {
      await payload.update({
        id: result.id,
        collection: slug,
        data: {
          _verified: true,
        },
      })
    }
```

i think it should be like this 

```
 if (verify) {
      await payload.update({
        id: result.id,
        collection: slug,
        data: {
          _verified: true,
        },
       req,
      })
    }
```

### Payload Version

2.0.0

**Comment by jmikrut** [comment]
Hey @PatrikKozak — the notes from @geminigeek look right to me, and there's a PR open! Beautiful! Take a look tomorrow, will ya?

**Comment by DanRibbens** [comment]
Thanks @geminigeek!
Will be released in 2.0.9
